<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>PreferencesDialogue</class>
    <widget class="QDialog" name="PreferencesDialogue">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>347</width>
                <height>289</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="windowTitle">
            <string>Preferences</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QTabWidget" name="tabWidget">
                    <property name="currentIndex">
                        <number>0</number>
                    </property>
                    <widget class="QWidget" name="localTab">
                        <attribute name="title">
                            <string>Local</string>
                        </attribute>
                        <layout class="QFormLayout" name="formLayout">
                            <item row="0" column="0">
                                <widget class="QLabel" name="zoomFactorLabel">
                                    <property name="text">
                                        <string>Default zoom factor</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QDoubleSpinBox" name="zoomFactorSpinBox">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="minimum">
                                        <double>0.250000000000000</double>
                                    </property>
                                    <property name="maximum">
                                        <double>5.000000000000000</double>
                                    </property>
                                    <property name="singleStep">
                                        <double>0.100000000000000</double>
                                    </property>
                                    <property name="value">
                                        <double>1.000000000000000</double>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="themeLabel">
                                    <property name="text">
                                        <string>Article view theme</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QComboBox" name="themeComboBox">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="remoteTab">
                        <attribute name="title">
                            <string>Remote</string>
                        </attribute>
                        <layout class="QGridLayout" name="gridLayout">
                            <item row="0" column="0">
                                <widget class="QPushButton" name="exportHistoryButton">
                                    <property name="text">
                                        <string>Export history</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="historySizeLabel">
                                    <property name="text">
                                        <string>History Size</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QSpinBox" name="historySizeSpinBox">
                                    <property name="maximum">
                                        <number>32767</number>
                                    </property>
                                    <property name="singleStep">
                                        <number>10</number>
                                    </property>
                                    <property name="value">
                                        <number>100</number>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="0">
                                <widget class="QLabel" name="suggestionSizeLabel">
                                    <property name="text">
                                        <string>Suggestion Size</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="1">
                                <widget class="QSpinBox" name="suggestionSizeSpinBox">
                                    <property name="minimum">
                                        <number>1</number>
                                    </property>
                                    <property name="maximum">
                                        <number>99</number>
                                    </property>
                                    <property name="value">
                                        <number>10</number>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QPushButton" name="clearHistoryButton">
                                    <property name="text">
                                        <string>Clear history</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="4" column="0" colspan="2">
                                <widget class="QPushButton" name="createNgramButton">
                                    <property name="text">
                                        <string>Create n-gram index</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>PreferencesDialogue</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>PreferencesDialogue</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
